README.md

```diff
@@ -0,0 +1,34 @@
+# FeedHenry RainCatcher Simple Store
+
+A CRUD storage engine that supports both in-memory and $fh.db backends.
+
+# Usage
+
+The initial config requires an entity name and the initial seed data:
+
+```javascript
+const Store = require('fh-wfm-simple-store');
+const store = new Store('user');
+store.init({ ... }).then(function() {
+  // store is available for use
+});
+```
+
+You can additionally utilize the `listen` method to have the story listen to conventially-named mediator topics.
+
+```javascript
+const store = new Store('user');
+store.init({}).then(function() {
+  store.listen('', mediator);
+  mediator.request('wfm:user:list').then(console.log);
+  // the above pattern is also available through the `topics` property:
+  store.topics.request('list').then(console.log);
+})
+```
+
+# DB Api
+The `$fh.db` backend utilizes the storage engine provided by the FeedHenry Cloud API, wrapped by the Simple Store API and is activated when the `WFM_USE_MEMORY_STORE` enviroment variable is set to `false`. The general configuration is done via environment variables.
```

Comment:
I think it should use $fh.db by default.

If a developer wants the in-memory version then they should set an  environment variable for this.

Otherwise, users could lose data if they assume it is using storage out of the box.